_config.yml

```diff
@@ -16,3 +16,5 @@ plugins:
   - jekyll-seo-tag
 
 search_enabled: true
+
+source: ./geckoview
```

Comment:
Mild preference of `docs` over `geckoview` since everything in this package is related to GeckoView. But nbd.